setup.py

```diff
@@ -68,34 +67,31 @@ def package_files(directory, strip_leading):
               'pytest-cov',
               'codecov'
           ],
+          'ci': ['codecov'],
+          'tf': ['tensorflow>=2.2.0'],
```

Comment:
`tf` is already in the conda envs, so will be installed on the PC already. We need this bit rather in the `[pi]` section, but there it won't be working as there is no `tf>1.15` in PyPI.

Reply:
PyPy is only one of places that we look for. PiWheels has `2.3.1` in the package repo. This is the correct thing to do to support Pip and regular `setup.py` methods to install the package. Not everyone uses `conda` / `miniconda`.

Reply:
Actually if you run
```
pip install tensorflow>=2.2.0
```
on RPi you get version 1.14.0 installed.

Reply:
Hmm.. That is super strange.. Try `tensorflow=2.2.0` ?

Reply:
I think we should be installing via conda on the PC, this is much easier and avoids any problems with incompatible packages, etc. This was already in the docs. On Pi/Nano, etc, where there is no conda, we need to use PyPi. Let me check PiWheels...

Reply:
`conda` is too magical. It really messes with virtual environments when you are dealing with cuDNN and other shared libraries. It _must_ be optional.

Reply:
Are you running `pip install tensorflow>=2.2.0` on a PC on a Pi or someplace else ? Are you sure its looking for piwheels ?

Reply:
Actually I already did this, PiWheels is part of the standard place to look for, I already pasted that in the other PR:
```
(env4) pi@donkeypi:~/projects/donkeycar $ pip install tensorflow==
Looking in indexes: https://pypi.org/simple, https://www.piwheels.org/simple
ERROR: Could not find a version that satisfies the requirement tensorflow== (from versions: 0.11.0, 1.12.0, 1.13.1, 1.14.0)
```

Reply:
You can also try `pip install tensorflow == 2.3.1` on a Pi.

Reply:
This doesn't work as the only packages on armv7 in pypi and pinwheels are `0.11.0, 1.12.0, 1.13.1, 1.14.0`.

Reply:
Your change here is fine. It won't resolve the install on PI, but it won't harm anything if you have started w/ a Conda env and run it on top, so looks safe, and makes sense if you want to run w/o conda, good point.